app/models/katello/glue/pulp/repo.rb

```diff
@@ -361,16 +361,10 @@ def full_path(smart_proxy = nil, force_https = false)
       scheme   = (self.unprotected && !force_https) ? 'http' : 'https'
       if docker?
         "#{pulp_uri.host.downcase}/#{container_repository_name}"
-      elsif file?
-        "#{scheme}://#{pulp_uri.host.downcase}/pulp/isos/#{relative_path}/"
       elsif ostree?
-        "#{scheme}://#{pulp_uri.host.downcase}/pulp/ostree/web/#{relative_path}"
-      elsif deb?
-        "#{scheme}://#{pulp_uri.host.downcase}/pulp/deb/#{relative_path}/"
-      elsif ansible_collection?
-        "#{scheme}://#{pulp_uri.host.downcase}/pulp/content/#{relative_path}/"
+        "#{scheme}://#{pulp_uri.host.downcase}/pulp/content/web/#{relative_path}"
```

Comment:
@sjha4 would it make sense to show a url of   `https://HOSTNAME/pulp_ansible/galaxy/#{relative_path}/`  here?

Reply:
Yes..That makes sense..👍🏽 
Can we change it to `https://HOSTNAME/pulp_ansible/galaxy/#{relative_path}/api `